docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -293,7 +293,6 @@ def upload_user_avatar(
         file,
         key,
         {
-            "ACL": "public-read",
```

Comment:
Ah, this one is more special. Is it the same django view-handling case as for the project thumbnail?

Reply:
Avatars are handled by [PublicFilesView](https://github.com/opengisch/qfieldcloud/blob/5fd1f5c338efd34c9b800760c6cda389bbdd1429/docker-app/qfieldcloud/core/views/files_views.py#L387)

Project thumbnails are handled by [ProjectMetafilesView](https://github.com/opengisch/qfieldcloud/blob/5fd1f5c338efd34c9b800760c6cda389bbdd1429/docker-app/qfieldcloud/core/views/files_views.py#L367C7-L367C27)

In any case, since the other s3 update where we use `s3.generate_presigned_url()`, "ACL": "public-read", is definitely no longer needed.